swagger-java-zenodo-client/README.md

```diff
@@ -0,0 +1 @@
+Zenodo does not provide an openAPI or Swagger yaml. You can find a list of the Zenodo REST APIs at this endpoint: https://zenodo.org/api/ and you can find the REST API documentation here: http://developers.zenodo.org/
```

Comment:
That was my mistake, a long time ago I created https://github.com/dockstore/zenodo-client/blob/master/swagger.yaml as a demo but neglected to show where I found it. 
Since the host says sandbox.zenodo.org, I wonder if I found it somewhere in their staging environment

Reply:
I would add something like "at one point there was a swagger.yaml in their staging environment that we have a copy of here..." in case some day the Zenodo API changes and we need to modify the Swagger, but I can see that as being overkill.